admin/load.php

```diff
@@ -170,6 +171,8 @@ function perflab_render_modules_page_field( $module_slug, $module_data, $module_
 					);
 				} elseif ( 'database/sqlite' === $module_slug && ! class_exists( 'SQLite3' ) ) {
 					esc_html_e( 'The SQLite module cannot be activated because the SQLite extension is not loaded.', 'performance-lab' );
+				} elseif ( is_wp_error( $can_load_module ) ) {
+					esc_html_e( 'The module cannot be managed with Performance Lab since it is already active as a standalone plugin.', 'performance-lab' );
```

Comment:
This condition should be the very first one in the list, as it is the most specific ones. All other messages here are fallbacks for when there is no `WP_Error` being returned.

Also, please don't hardcode the message here, as that misses the point of using a `WP_Error`. We should display `$can_load_module->get_error_message()` here.